blocks/extensions/__init__.py

```diff
@@ -128,6 +128,24 @@ def on_interrupt(self):
         pass
 
 
+class CallbackName(str):
```

Comment:
I preferred `Callback`, considering that a callback in our case is nothing but a string anyway, adding the `Name` is redundant IMHO and just seems to suggest that there is an actual `Callback` class floating around somewhere, which there isn't.